aws/aws_image.go

```diff
@@ -322,6 +330,15 @@ func getArchitecture(flavor string) string {
 	return "x86_64"
 }
 
+func isGravitonProcessor(flavor string) bool {
+	if flavor == "" {
+		return false
+	}
+
+	_, isExist := GravitonProcessors[strings.ToLower(strings.Split(flavor, ".")[0])]
```

Comment:
is there a flavor with a period in it? it doesn't look like it? do we need this split here?

Reply:
yes, the flavor has a period, we need split here for GravitonProcessors mapping like this: https://github.com/nanovms/ops/blob/a69fa8582afb15b16156c1fe1e629ecd4708dc71/aws/aws_image.go#L308
example: r6g.medium, a1.xlarge